itest/lnd_onion_message_test.go

```diff
@@ -0,0 +1,75 @@
+package itest
+
+import (
+	"time"
+
+	"github.com/btcsuite/btcd/btcec/v2"
+	"github.com/lightningnetwork/lnd/lnrpc"
+	"github.com/lightningnetwork/lnd/lntest"
+	"github.com/stretchr/testify/require"
+)
+
+// testOnionMessage tests sending and receiving of the onion message type.
+func testOnionMessage(ht *lntest.HarnessTest) {
+	alice := ht.NewNode("Alice", nil)
+	bob := ht.NewNode("Bob", nil)
```

Comment:
How about we make this a multi-hop path? So at least 3 nodes, then we make sure messages can be sent in both directions.

Reply:
Yeah, correct. We'll test multi-hop extensively in follow-up PR.